### DISCUSSION TITLE
Unable to add EKS cluster to Teleport Cloud

### DISCUSSION THREAD

**Discussion by along45** [gravitational/teleport#34399] [Q&A]
I'm using this article (https://goteleport.com/docs/kubernetes-access/getting-started/) and follow the procedures as described in the document. 
However after helm chart install the pod keeps on restarting constantly - it doesn't generate any logs and I can't use describe. 

I'm using version 14.1.1
Teleport v14.1.1 git:v14.1.1-0-gfb6429e go1.21.3

Any ideas what might happen? Thanks!

**Comment by dvillarraga** [comment]
Are you reinstalling teleport? The first time worked for me, I had to remove all DynamoDB tables to install it again, you might need to change the chart setup if you plan to install it with existing Buckets/DynamoDB Tables.

**Comment by webvictim** [comment]
Please share the Helm values you're using.
